source/infrastructure/free-radius/debug.html.md.erb

```diff
@@ -23,13 +23,13 @@ It will contain all the details of the authentication request which can be used
 
 ## Mailing List
 
-You can get help from the FreeRADIUS community by asking questions on their [user mailing list][mailing list link].
+You can get help from the FreeRADIUS community by asking questions on their [user mailing list]((https://lists.freeradius.org/mailman/listinfo/freeradius-users)).
 
 You will need to sign up before you can ask questions or gain access to the archives.
 
 ## eapol_test
 
-In order to simulate UDP requests locally, there is a tool called [eapol_test][eapol test link].
+In order to simulate UDP requests locally, there is a tool called [eapol_test](https://manpages.debian.org/testing/eapoltest/eapol_test.8.en.html).
```

Comment:
These links shouldn't be missing, as they do look to be defined at the bottom of the file?

Reply:
It's now linking to an image I think, is that your intent?